kuberneteth.yaml

```diff
@@ -37,10 +46,13 @@ nodes:
 # ...
 monitor:
   name: monitor
+  # type can be [ClusterIP, NodePort, LoadBalancer]
+  serviceType: LoadBalancer
```

Comment:
I would keep `ClusterIP` default here, as it works on all environments

Reply:
Ah yes, I must have missed that when trying to set everything back from my configuration